bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -217,6 +217,25 @@
 			box-shadow: none;
 			-webkit-box-shadow: none;
 		}
+		.mdl-textfield {
+			&__input,
+			&__label {
+				font-family: "Helvetica","Arial",sans-serif;
+				font-size: 14px;
+				text-align: right;
+			}
+			&__input {
+				font-weight: 700;
+			}
+			&__label {
+				@media screen and (max-width: $layout-tablet-size-threshold) {
+					margin: 0 -12px;
+					padding: 0 12px 0 0;
+				}
+				margin: 0 -16px;
+				padding: 0 16px 0 0;
```

Comment:
I am not expert but I think you should probably reverse the order, that is first set margin and padding and finally the `@media`

Reply:
Not an expert myself, but I believe the more specific definition takes precedence, whatever the sequence in the style sheet file. I did try playing with screen sizes in the browser debug tool.

Reply:
But if you see how it is done in this file, you will see that this is always done in the reverse order.
Maybe it is not important, but just for consistency for the whole file, I would prefer the reverse order.